enterprise_access/apps/bffs/serializers.py

```diff
@@ -261,7 +263,7 @@ class EnterpriseCourseEnrollmentSerializer(BaseBffSerializer):
     org_name = serializers.CharField()
     course_run_status = serializers.CharField()
     display_name = serializers.CharField()
-    emails_enabled = serializers.BooleanField()
+    emails_enabled = serializers.BooleanField(required=False, allow_null=True)
```

Comment:
Good catch that this is valid to be nullable! ([source](https://github.com/openedx/edx-platform/blob/139b4167b37b49d2d69cccdbd19d8ccef40d3374/lms/djangoapps/bulk_email/api.py#L45))